### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by sa35577**
Merged the 2 app.tsx's, placed my files in the relevant directories.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
+    "start": "concurrently \"node index.js\" \"expo start --ios\"",
```
Comment (haleysong576) [inline package.json]: why was this change needed? now the default script is running for ios setting and it requires XCode to be installed

File: src/screens/login.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import { StatusBar } from 'expo-status-bar';
+import {Alert, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {useState} from "react";
+import { NavigationContainer} from "@react-navigation/native";
+import {createReactNativeMatcher} from "@expo/metro-config/build/transformer/createMatcher";
+
```
Comment (haleysong576) [inline src/screens/login.tsx]: Can you add "import React from 'react';" along with the other imports? I'm getting errors without it

File: src/screens/signup.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
+import { StatusBar } from 'expo-status-bar';
+import {Alert, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {useEffect, useState} from "react";
+import axios from "axios";
+
```
Comment (haleysong576) [inline src/screens/signup.tsx]: Also add "import React from 'react';" here

File: src/screens/login.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    axios({
+      url: "http://localhost:3000/api/user/login",
+      method: "POST",
+      data: {
+        email: uName,
+        password: pWord
+      }
+    }).then((response:string) => {
+      // console.log(response);
+      console.log("successful auth");
```
Comment (haleysong576) [inline src/screens/login.tsx]: Can we navigate to classroom finder page when there's successful login? (we can navigate to the main search page when it's ready)

File: App.tsx [hunk 5]
```diff
@@ -1,57 +1,56 @@
...
-       screenOptions={{
-        header: () => <Header />
-      }}>
-        <Stack.Screen 
-          name = "classroomFinderScreen"
-          component={Classroom}/>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName={"Login"} screenOptions={{header: () => <Header />}}>
+        <Stack.Screen name={"Login"} component={LoginScreen} options={{title: 'Login', headerShown:false}} />
+        <Stack.Screen name={"Signup"} component={SignupScreen} options={{title: 'Signup', headerShown: false}} />
```
Comment (haleysong576) [inline App.tsx]: To navigate to classroom finder screen we need to keep the Stack.Screen that has Classroom component

**Review by haleysong576 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sa35577) [reply package.json]: start will now link to web, I used this command because I needed some way to start the backend temporarily and then run the frontend

File: client/src/screens/classroomFinder.tsx [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import * as React from 'react'
-import { useState } from 'react';
+import {useEffect, useState} from 'react';
```
Comment (victorjzsun) [inline client/src/screens/classroomFinder.tsx]: Is this change necessary?

File: client/src/screens/classroomFinder.tsx [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
 import { Dimensions, StyleSheet, Text, View } from 'react-native';
 import MapView from 'react-native-maps';
 
 
 
 const ClassroomFinderScreen = () => {
+
```
Comment (victorjzsun) [inline client/src/screens/classroomFinder.tsx]: ditto

**Review by victorjzsun — APPROVED** [review]
lgtm?
